play-runtime/src/main/scala/play/grpc/internal/PlayRouter.scala

```diff
@@ -20,6 +20,7 @@ import scala.compat.java8.FutureConverters._
  * INTERNAL API
  */
 @InternalApi private[grpc] object PlayRouterHelper {
+  @deprecated("Prefer handlerFunction(akka.japi.function.Function[])", "0.10.0")
```

Comment:
Tentatively decided the next version is a `0.10.0` but maybe we should bump it to `0.30.0` to leave a big gap from the current `0.9.1`.

Bumping to akka-grpc 1.1.1 means we're using Akka HTTP 10.2.3+, reason why I think a big version bump could be useful.